config_example.sh

```diff
@@ -53,3 +53,15 @@ set -x
 
 # External subnet
 #export EXTERNAL_SUBNET="11.0.0.0/24"
+
+# Enable testing of custom machine-api-operator-image
+#export TEST_CUSTOM_MAO=true
+
+# Custom machine-api-operator image with tag
+#export CUSTOM_MAO_IMAGE="quay.io/mao-user/machine-api-operator:mao-fix"
+
+# Repository that is holding any custom machine-api-operator changes
+#export REPO_NAME="mao-user"
+
+# Name os branch in the above repo which contains the custom MAO changes
```

Comment:
Typo s/os/of

Also should we say "Git repository" here to differentiate from the image repository above?